cli/types/cypress.d.ts

```diff
@@ -1475,6 +1484,13 @@ declare namespace Cypress {
      */
     not(selector: string, options?: Partial<Loggable & Timeoutable>): Chainable<JQuery>
 
+    /**
+     * Invoke a command synchronously, without using the command queue.
+     *
+     * @see https://on.cypress.io/custom-queries
+     */
+    now(name: string, ...args: any[]): Promise<any> | ((subject: any) => any)
```

Comment:
I tried to be more specific, with `name`, `args` and the return type mapping to `ChainableMethods`, but typescript ended up caught in a recursion, since `now` is one of those `ChainableMethods`.

What we really need to do is go through and separate out "things on `cy` (like now, off, on, once)" from "commands that can be chained (like click, get, scrollTo)", but that would involve a larger refactor of the types than I want to undertake in this PR.